akka-typed/src/main/scala/akka/typed/Ask.scala

```diff
@@ -59,6 +61,10 @@ object AskPattern {
         val b = adapter.ActorRefAdapter[U](a)
         (b, a.result.future.asInstanceOf[Future[U]], a)
       }
+
+    val ref: ActorRef[U] = _ref
+    val future: Future[U] = _future
```

Comment:
Yes, but then `_promiseRef` couldn't be `private[this]` (since `ref` and `future` need to be public).